specs/DotNetProjectFile.Analyzers.Specs/Extensions/Microsoft.CodeAnalysis.Diagnostics.DiagnosticAnalyzer.cs

```diff
@@ -0,0 +1,26 @@
+﻿using CodeAnalysis.TestTools;
+using CodeAnalysis.TestTools.Contexts;
+
+namespace Microsoft.CodeAnalysis.Diagnostics;
+
+internal static class DiagnosticAnalyzerExtensions
+{
+    public static ProjectAnalyzerVerifyContext ForProject(this DiagnosticAnalyzer analyzer, string fileName)
+    {
+        var name = Path.GetFileNameWithoutExtension(fileName);
+        var file = new FileInfo($"../../../../../projects/{name}/{fileName}proj");
```

Comment:
Does this mean that the fileName always has the .cs/.vb extension? What happens if it is called with another extension?